### PR TITLE
Conditional statements/ it-academy.by / M-JC1-66-20 / Anton Khramau / Done Task 1-14

### PR DESCRIPTION
**Pull request by Igrok023451**
## Pull request state

Please indicate actual status for PR:
- [x] on review
- [ ] reviewed
- [x] fixed
- [ ] on approve
- [x] approved

### CODE REVIEW

**Comment by Igrok023451** [comment]
Второй 6-ой это 7-мь.

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task05.java [hunk 1]
```diff
@@ -34,7 +36,34 @@ public static void main(String[] args) {
...
+                return Month.SEPTEMBER.toString();
+            case 10:
+                return Month.OCTOBER.toString();
+            case 11:
+                return Month.NOVEMBER.toString();
+            case 12:
+                return Month.DECEMBER.toString();
+            default:
+                return null;
+        }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task05.java:67]: What is it?

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task09.java [hunk 2]
```diff
@@ -34,6 +34,7 @@ public static void main(String[] args) {
     static int getPorchNumber(int numberFloors, int numberApartmentsPerFloor, int apartmentNumber) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+         return (int)Math.ceil(apartmentNumber / (double)(numberFloors * numberApartmentsPerFloor));
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task09.java:37]: What is it?

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java [hunk 3]
```diff
@@ -40,6 +43,15 @@ public static void main(String[] args) {
     static String getSymbolType(char symbol) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        if(Pattern.matches("[a-zA-Z]",String.valueOf(symbol) )){
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java:46]: What is it?

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java [hunk 4]
```diff
@@ -40,6 +43,15 @@ public static void main(String[] args) {
...
+            return "latin";
+        }
+        if(Pattern.matches("[а-яА-Я]",String.valueOf(symbol) )){
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java:49]: What is it?

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java [hunk 5]
```diff
@@ -40,6 +43,15 @@ public static void main(String[] args) {
...
+            return "cyrillic";
+        }
+        if(Pattern.matches("[0-9]",String.valueOf(symbol) )){
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task11.java:52]: What is it?

**Review by rakovets — CHANGES_REQUESTED** [review]
How to work with Java APIs used in solutions?

Tests passed: 128/140.
Fix it.

**Review by rakovets — COMMENTED** [review]
Approved.

**Pull request closed without merging**